frontend/www/js/omegaup/components/problem/FilterDifficulty.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="card mt-4">
+    <div class="card-body">
+      <h3>{{ T.wordsDifficulty }}</h3>
+      <div
+        v-for="(difficulty, index) in difficulties"
+        :key="index"
+        class="form-check"
+      >
+        <label class="form-check-label">
+          <input
+            class="form-check-input"
+            type="radio"
+            name="difficulty"
+            :value="difficulty"
+            @change="$emit('change', $event.target.value)"
```

Comment:
nota que si el componente padre provee un valor default para la dificultad, esto no va a funcionar porque [el `<input>` debería tener un `v-model`](https://vuejs.org/v2/guide/forms.html#Radio). entonces hay que hacer:

```suggestion
            v-model="currentDifficulty"
```

y luego abajo

```typescript
currentDifficulty = this.selectedDifficulty;

@Watch('selectedDifficulty')
onSelectedDifficultyChanged(val: string | null) {
  this.currentDifficulty = val;
}

@Emit('change')
@Watch('currentDifficulty')
onCurrentDifficultyChanged(val: string | null) {
  return val;
}
```